tests/lib/Stubs/ItemGroupList.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzRecommendationClient\Tests\Stubs;
+
+use ArrayIterator;
+use Ibexa\Contracts\Personalization\Value\ItemGroupListInterface;
+use Traversable;
+
+final class ItemGroupList implements ItemGroupListInterface
```

Comment:
Needed for `Traversable` - implementing class needs to be told which variant of iteration it supports.
```suggestion
final class ItemGroupList implements \IteratorAggregate, ItemGroupListInterface
```